drivers/usb/fotg210/fotg210-hcd.c

```diff
@@ -5541,8 +5541,15 @@ static void fotg210_init(struct fotg210_hcd *fotg210)
 {
 	u32 value;
 
+#ifdef SOC_BOUFFALOLAB
```

Comment:
These should be quirks instead set either based on compatible string, or separate fdt properties.